release notes/v0.38.0.md

```diff
@@ -0,0 +1,176 @@
+* k6 will now detect invalid thresholds:
+    ```javascript
+    export const options = {
+      // ...    
+      thresholds: {
+        // Incorrect thresholds expressions:
+        http_req_failed: ['rave<0.01'],  // e.g. "rave" is not a valid aggregation method
+        // Thresholds applying to a non-existing metrics:
+        iDoNotExist: ['p(95)<200'], // e.g. the metric 'iDoNotExist' does not exist
+        // Thresholds applying an aggregation method that's unsupported by the metric they apply to:          
+        my_counter: ['p(95)<200'], // Counter metrics do not support the p() aggregation method
+     },
+    };
+    ```
+
+### Disabling colors ([#2410](https://github.com/grafana/k6/pull/2410))
+
+In addition to the [`--no-color` CLI flag](https://k6.io/docs/using-k6/options/#no-color), the ANSI color escape codes emitted by k6 can now also be disabled by setting the `NO_COLOR` or `K6_NO_COLOR` environment variables, following the [NO_COLOR standard](https://no-color.org/).
+
+```bash
+# No color output
+K6_NO_COLOR=true k6 run script.js
+
+# No color output
+NO_COLOR= k6 run script.js
+```
+
+### Support for encrypted TLS private keys
+
+You can now use passphrase-protected private keys when authenticating with TLS. Using the `password` property of an options' `tlsAuth` object, you can now indicate the passphrase to use to decrypt a private key. Note that this support is limited to the scope of [RFC1423](https://datatracker.ietf.org/doc/html/rfc1423) and does not support PKCS8 keys, as they're not yet supported by the Golang standard library. 
+
+```javascript
+export const options = {
+  tlsAuth: [
+    {
+      domains: ['example.com'],
+      cert: open('mycert.pem'),
+      key: open('mycert-key.pem'),
+      password: 'mycert-passphrase',
+    },
+  ],
+};
+```
+
+Thanks, @Gabrielopesantos, for the [contribution](https://github.com/grafana/k6/pull/2488).
+
+### Improve JSON output's performance ([#2436](https://github.com/grafana/k6/pull/2436))
+
+The JSON output was optimized and now should be around 2x as more performant at outputting metrics.  This means that it either will be able to export twice as many metrics and use half the resources to do the same amount of metrics. As a side effect, there is a slight breaking change in which the `tags` field is no longer sorted, while it was previously.
+
+### Treat panics as interrupt errors ([#2453](https://github.com/grafana/k6/pull/2453))
+
+We changed the behavior of how k6 treats Go panics, which may happen because of bugs in k6 or in a JavaScript k6 extension. Previously, the behavior was to catch the panic and log it as an error. Starting with **v0.38.0**, whenever k6 observes a Go panic, it logs an error like before, but more importantly, it will abort the script execution and cause k6 to exit with a non-0 exit code. This will help extension authors to identify issues in their extensions more easily. 
```

Comment:
```suggestion
We changed the behavior of how k6 treats Go panics, which may happen because of bugs in k6 or in a JavaScript k6 extension. Previously, the behavior was to catch the panic and log it as an error.

Starting with **v0.38.0**, whenever k6 observes a Go panic, it logs an error like before, but more importantly, it will abort the script execution and cause k6 to exit with a non-0 exit code. This will help extension authors to identify issues in their extensions more easily. 
```